--- a/test/dialect/mysql/test_reflection.py
+++ b/test/dialect/mysql/test_reflection.py
@@ -301,7 +301,6 @@
                 mariadb_auto_increment="5",
                 mariadb_avg_row_length="3",
                 mariadb_password="secret",
-                mariadb_connection="fish",
             )
         else:
             kwargs = dict(
@@ -322,7 +321,6 @@
             assert def_table.kwargs["mariadb_auto_increment"] == "5"
             assert def_table.kwargs["mariadb_avg_row_length"] == "3"
             assert def_table.kwargs["mariadb_password"] == "secret"
-            assert def_table.kwargs["mariadb_connection"] == "fish"
 
             assert reflected.kwargs["mariadb_engine"] == "MEMORY"
 
@@ -331,7 +329,6 @@
             assert reflected.kwargs["mariadb_default charset"] == "utf8mb4"
             assert reflected.kwargs["mariadb_avg_row_length"] == "3"
-            assert reflected.kwargs["mariadb_connection"] == "fish"
 
             # This field doesn't seem to be returned by mariadb itself.
             # assert reflected.kwargs['mariadb_password'] == 'secret'
